Source: written by Calvin Walton <calvin.walton@kepstin.ca>
Upstream: https://github.com/drkjam/netaddr/pull/72
Reason: Ensure the test runner returns an error code when tests fail
diff -u -r netaddr-0.7.11.orig/netaddr/tests/__init__.py netaddr-0.7.11/netaddr/tests/__init__.py
--- netaddr-0.7.11.orig/netaddr/tests/__init__.py	2014-06-29 19:43:45.225228400 -0400
+++ netaddr-0.7.11/netaddr/tests/__init__.py	2014-06-29 20:32:51.188987000 -0400
@@ -65,8 +65,9 @@
 #-----------------------------------------------------------------------------
 def run():
     runner = unittest.TextTestRunner()
-    runner.run(test_suite_all())
+    return runner.run(test_suite_all())
 
 #-----------------------------------------------------------------------------
 if __name__ == "__main__":
-    run()
+    result = run()
+    sys.exit(not result.wasSuccessful())
